include/deal.II/distributed/shared_tria.h

```diff
@@ -106,6 +113,39 @@ namespace parallel
                                          const std::vector< CellData< dim > > &cells,
                                          const SubCellData &subcelldata);
 
+      /**
+       * Return true_subdomain_ids .
```

Comment:
Please expand on this. One shouldn't ask readers to go look up a private member variable to know what the function does.

How about this:
```
  Return a vector of length Triangulation::n_active_cells() where each element stores the subdomain id of the owner of this cell. The elements of the vector are obviously the same as the subdomain ids for locally owned and ghost cells, but are also correct for artificial cells that do not store who the owner of the cell is in their subdomain_id field.
```